.github/issue_template.md

```diff
@@ -1,11 +1,19 @@
 ## How to file an issue - REMOVE THIS SECTION
- - Add a prefix to the title scoping the issue, e.g. `gui: `, `player: ` ... 
- - use a title that is clear enough
- - If appropriate, add a topic label to group the issue with a larger effort and allow filtering by that topic.
 
-## How to file a bug - REMOVE THIS LINE
+This repository is only for Plex Media Player (PMP). Please do not open issues related to other Plex clients (Roku, Apple TV, etc) or Plex Media Server.
+
+Each filed issue must please:
+
+  - include a concise description of the issue and steps to reproduce.
+  - include a sample media file or media file information when applicable.
+    - Generating samples: https://support.plex.tv/articles/201035968-generating-sample-files-from-media/.
+    - Retrieving XML media information: https://support.plex.tv/articles/201998867-investigate-media-information-and-formats/. Please attach within zip.
+  - be accompanied by _clean_ logs.
+    - PMP logs can be found here: https://support.plex.tv/articles/207338748-plex-media-player-logs/
+    - Clean log instructions: Clear your logs; reproduce the issue; attach logs in a .zip file.
 
 ### Test environment
+
 **PMS Version:**
 **PMP Version:**
 **Platform (Windows/OSX/Embedded):**
```

Comment:
Do we want to specify other platforms?  Perhaps: `(Windows/OSX/Embedded Intel/Embedded RPi/Linux AppImage)`?

Reply:
I'll add in the the embedded distinctions. I'm hesitant to include app image. Issue reports from an app image maintainer is totally okay with me although I'm concerned we'd currently have challenges supporting app image installations.